### ISSUE TITLE
Empty .psmcfa File

### ISSUE THREAD

**Issue by Rashed-shou** [lh3/psmc#61]
Hello Developer, 
I polished my nanopore raw genome sequence data with HiC data of the same species. My analysis achieved a BUSCO completeness of 96.6%. However, I tried the PSMC tools several times with either masked or unmasked genomes combined with the resequencing or HiC data, but I did not obtain any plot. Unfortunately, the output diploid.psmcfa file is empty (0 kb) in every time. In this step, I used -q20.  Could you please help me troubleshoot the issue?
![PSMC output](https://github.com/user-attachments/assets/e7aa3bfa-ef53-4262-a030-b6259b9203b7)

**Comment by timurrxxcd** [comment]
Hi, did you find solution? I have same issue.

**Comment by Rashed-shou** [comment]
No, I didn't still find any solution and no reply from the developer!!

**Comment by timurrxxcd** [comment]
I understood the reason. bcz it depends on quality of bam file, where it's well mapped or variants well called or not. Thus you need to choose best quality bam files. or if you dont' have you can make lower -C50 to -C40 and -d 10 -D 100 to -d 8 -D 120. it may help somehow, but remember as you make lower these parameters, your results will be more affected leading to unbeliavable results. Good luck brother!

**Comment by chichizhao** [comment]
Recently, I got the same problem, here, I believe it may the -D problem in your data. As in my data, i set -D into 200, it still have the same problem, with empty psmcfa file. after i carefully check the psmc.fq.gz file, and the script with the vcf2fq function and the fq2psmcfa script, i got this ideal in my data. There are too many low character base in the psmc.fq.gz file, which means in "N"  in the transition from fastq to fastq in the psmc analysis workflow.  I accidently find that, the  -D is the big things, my bam file has 53 GB, like yours you got 44GB, and the average depth of my is the 680 x. Therefore, for my data, there are too may data are consider to be think as the low quilety sites. here i swith the -D into 1400, two times over the avarage depth, here, i believe it may solve the problem, you can check the -D and set it properlly.
